helix-dap/src/types.rs

```diff
@@ -840,7 +870,7 @@ pub mod events {
     #[serde(rename_all = "camelCase")]
     pub struct TerminatedBody {
         #[serde(skip_serializing_if = "Option::is_none")]
-        pub restart: Option<Value>,
+        pub restart: Option<bool>,
```

Comment:
This can be an `any` according to the spec so it should stay as `Value` https://microsoft.github.io/debug-adapter-protocol//specification.html#Events_Terminated

It sounds like we shouldn't be interpreting this argument and instead should be passing it as-is to `Client::launch` or `Client::attach` if it is `Value::Bool(true)` or `Value::Object(_)`

Reply:
Yes you’re right, I will change this

Reply:
I think i was confusing this with disconnect's restart argument which does need to be a boolean. This has been fixed now.